datasets/beheerkaart/cbs_grid/dataset.json

```diff
@@ -42,7 +49,7 @@
           },
           "cbsGeometrie100": {
             "$ref": "https://geojson.org/schema/Polygon.json",
-            "description": "Vlak-coördinaten van het BKT-CBS-grid-100-vlak."
+            "description": "Vlak-co\u00f6rdinaten van het BKT-CBS-grid-100-vlak."
```

Comment:
is dit nodig? :-o

Reply:
De `pre-commit` hooks doen dit.

Reply:
Dit is handig. Als alle bestanden ASCII zijn, zie je de NBSP's en soft hyphens sneller. Scheelt debuggen als die ergens een string match doen falen.